kasa/tests/test_bulb.py

```diff
@@ -171,7 +171,7 @@ async def test_non_variable_temp(dev: SmartBulb):
         await dev.set_color_temp(2700)
 
     with pytest.raises(SmartDeviceException):
-        dev.valid_temperature_range()
+        dev.valid_temperature_range
```

Comment:
The ruff issue will be fixed if you print the value like done below :-)

Reply:
All fixed!